### PR TITLE
add missing shards install before running specs in CI

### PR DESCRIPTION
**Pull request by jwoertink**
oops 😬

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -41,6 +41,8 @@ jobs:
     container: crystallang/crystal:${{ matrix.crystal_version }}-alpine
     steps:
       - uses: actions/checkout@v2
+      - name: Install shards
+        run: shards install
```
Comment (matthewmcgarvey) [inline .github/workflows/ci.yml]: It needs to go after the cache step or the cache will overwrite it and the shards will always install (worst of both worlds 😆)

**Review by matthewmcgarvey — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (jwoertink) [reply .github/workflows/ci.yml]: I copied it from Lucky https://github.com/luckyframework/lucky/blob/master/.github/workflows/ci.yml#L27-L34 so does that mean that's wrong too? 🤦

Comment (matthewmcgarvey) [reply .github/workflows/ci.yml]: Most definitely! We're real professionals haha

Comment (jwoertink) [reply .github/workflows/ci.yml]: Lovely! Looks like just about all of them are wrong 😂  Might as well start with this one.

**Pull request merged**